onnxruntime/core/session/standalone_op_invoker.cc

```diff
@@ -409,7 +410,7 @@ onnxruntime::Status CreateOp(const OrtKernelInfo* info,
   static FuncManager kFuncMgr;
   status = kernel_create_info->kernel_create_func(kFuncMgr, tmp_kernel_info, op_kernel);
   ORT_RETURN_IF_ERROR(status);
-  status = NodeRepo::GetInstance().AddNode(op_kernel.get(), std::move(node_ptr));
+  status = NodeRepo::GetInstance().AddNode(op_kernel.get(), std::move(node_ptr), arg_ptrs);
```

Comment:
std::move(arg_ptrs). This makes the intent crystal clear instead of silently moving it inside AddNode. Since move converts the l-value to r-value, AddNode signature also needs to be updated to accept a universal reference.

Reply:
Done.